che-tomcat8-slf4j-logback/src/assembly/bin/setenv.bat

```diff
@@ -17,7 +17,7 @@ if "%CHE_LOCAL_CONF_DIR%"==""   (
    exit
 )
 
-if "%JAVA_OPTS%"=="" (set JAVA_OPTS=-Xms256m -Xmx2048m -XX:MaxPermSize=256m -server)
```

Comment:
According to docs

>Turns Java VM features specific to server applications, such as sofisticated JIT compiler. Though this option is implicitely enabled for x64 virtual machines, it still makes sence to use it as according to documentation behaviour maybe changed in the future.

This parameter was come from the old ages there all our tomcats running in 32-bit VM on server. Now I think this is no longer needed. And I want to let JVM decide the best mode.